inst/rmarkdown/templates/mortality/skeleton/skeleton.Rmd

```diff
@@ -1,120 +1,236 @@
 
 ```{r}
 # Insert some code to generate a random number sequence
 ```
 
-### Cluster sampling
+#### Cluster sampling
 
 ```{r}
 # Insert some example code to do cluster sampling
 ```
 
-# Study Findings
+## Data collection
+
+## Data analysis
+<!-- TODO: add paragraph about R/sitrep -->
+
+# Results
 
 
 ```{r}
 # we first need to create a survery design that reflects the study
 suvery_design <- svydesign(
   ids = ~1, # no cluster ids
-  weights = NULL, #no sampling weights in this example
+  weights = NULL, # no sampling weights in this example
   data = study_data_cleaned
 )
 ```
 
-
-## Demographics
-
 ```{r}
-plot_age_pyramid(study_data_cleaned)
+# TODO: Arrivals and departures of included household members during the recall period;
 ```
 
-## Morbitity
 
-```{r}
-# symptoms start with sym_
+## Demographic information
 
-cols <- colnames(study_data_cleaned)
-symptom_columns <- cols[grepl(pattern = "^sym", x = cols)]
+TODO: add some sample text with inline R
 
-survey_ciprop_foreach_binary_variable(design = suvery_design, 
-                                      variables = symptom_columns) %>%
-  knitr::kable()
+Age distribution of current household population by five year age groups, dependency age groups, and by child and adult populations, by gender.
+```{r describe_by_age_group_and_sex}
+descriptive(study_data_cleaned, 
+            "age_group", 
+            "sex", 
+            rowtotals = TRUE, 
+            coltotals = TRUE) %>% 
+  rename("Age" = age_group, 
+         "Female cases (n)" = Female_n, 
+         "%" = Female_prop, 
+         "Male cases (n)" = Male_n, 
+         "%" = Male_prop) %>% 
+  kable(digits = 2)
 ```
 
+```{r describe_by_age_group_and_sex_wider_age_groups}
+study_data_cleaned <- study_data_cleaned %>% 
+  mutate(age_group_wide = age_categories(age_in_years, 
+                                         breakers = c(0, 1, 5, 15, 60)))
+
+descriptive(study_data_cleaned, "age_group_wide", "sex", rowtotals = TRUE) %>% 
+  # rename("Age" = age_group_wide, 
+  #        "Female cases (n)" = Female_n, 
+  #        "%" = Female_prop, 
+  #        "Male cases (n)" = Male_n, 
+  #        "%" = Male_prop) %>% 
+  setNames(c("Infant, child and adult populations", rep("", nrow(.)))) %>% # find better solution
+  kable(digits = 2)
+```
+
+Age distribution of current household population by five year age groups and by gender.
+```{r age_pyramid}
+plot_age_pyramid(study_data_cleaned, 
+                 age_group = "age_group", 
+                 split_by = "sex") + 
+  labs(x = "Cases (n)", y = "Age group (years)") +                    # change axis labels
+  theme(legend.position = "bottom", legend.title = element_blank())   # remove title and move legend
+```
+
+```{r}
+# TODO: Table - Age distribution of the sample population and of the study population.
+```
 
 
 ## Mortality
 
+Reported causes of death and cause-specific mortality rates (cause-specific mortality per 10000 still missing!)
+```{r}
+death_tbl <- function(x) {
```

Comment:
this section is a bit dense - worth making it in to a standalone function for the sitrep package?

Reply:
I also support making it a standalone function. I've added `case_fatality_rate_df()` to take care of most of this, but at the moment, I need to add a population option to take care of the situation you have here.

Reply:
this will be dealt with using the srvyr package so dw!